sdk/core/src/CommonNetworkMember/BaseNetworkMember/AnchoringService.ts

```diff
@@ -0,0 +1,80 @@
+import { ParsedOptions } from '../../shared/getOptionsFromEnvironment'
+import { extractSDKVersion } from '../../_helpers'
+import { ELEM_DID_METHOD, JOLO_DID_METHOD } from '../../_defaultConfig'
+import { RegistryApiService } from '@affinidi/internal-api-clients'
+import { KeysService } from '@affinidi/common'
+
+export class AnchoringService {
```

Comment:
It is not a part of `BaseNetworkMember` class hierarchy, and should belong to `services` directory.

Reply:
Actually just splitting of previous codebase, not sure where it should be as we will split this ugly class (1100 lines of code) in future to parts, so from my side it makes sense to keep it in one place. That is why I created a subdir in CommonNetworkMemeber for BaseNetworkMemeber

Reply:
Originally there was an ugly class called `CommonNetworkMember` with 3000 lines, it was split into smaller interdependent parts all put into `CommonNetworkMember` directory.

It's great that you decided to extract some more independent parts out of it in this PR. It does not make any sense to store these parts in `CommonNetworkMember` directory; they are independent after all. It does not make any sense at all to move `BaseNetworkMember` in its separate directory inside `CommonNetworkMember`.